src/msg/async/dpdk/DPDK.cc

```diff
@@ -562,7 +562,6 @@ bool DPDKQueuePair::init_rx_mbuf_pool()
     }
   }
 
```

Comment:
rte_mempool_count and rte_mempool_free_count are not defined in spdk/dpdk library, just remove it.